### PR TITLE
Smarts ppo agent

### PR DESCRIPTION
**Pull request by Dikshuy**
added changes required for training purposes

### CODE REVIEW

File: examples/single_agent.py [hunk 1]
```diff
@@ -50,10 +57,15 @@ def main(scenarios, sim_name, headless, num_episodes, seed, max_episode_steps=No
...
         ),
+<<<<<<< HEAD
         agent_builder=PPOPolicy,
         agent_params={"policy_params": adapter.policy_params},
         observation_adapter=adapter.observation_adapter,
         reward_adapter=adapter.reward_adapter,
+=======
+        # agent_builder=ChaseViaPointsAgent,
+        agent_builder=PPO,
+>>>>>>> f603a900507e2ba8579b3029d85408c197316b13
```
Comment (Adaickalavan) [inline examples/single_agent.py:68]: Please remove any merge conflicts from all the files. The above is an example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +23,8 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - Patched a restart of SUMO every 50 resets to avoid rampant memory growth.
 ## Removed
 - Removed `pview` from `make` as it refers to `.egg` file artifacts that we no longer keep around.
+=======
+>>>>>>> f603a900507e2ba8579b3029d85408c197316b13
```
Comment (Adaickalavan) [inline CHANGELOG.md:27]: Please rectify all merge conflicts.

File: examples/single_agent.py (hunk 1 above)
Comment (Dikshuy) [reply examples/single_agent.py]: have some reformating issue. will try to resolve it.

Comment (Dikshuy) [reply examples/single_agent.py]: removed all the merge conflicts now

Comment (christianjans) [reply examples/single_agent.py]: Hi @Dikshuy, I think there are still a couple more in the CHANGELOG.md... Although, I think the only files that really need to be changed are examples/single_agent.py and ultra/ultra/baselines/adapter.py, is this correct?

Comment (Dikshuy) [reply examples/single_agent.py]: yeah, only  `examples/single_agent.py` and `ultra/ultra/baselines/adapter.py` need to be changed everything else is the same.

Comment (christianjans) [reply examples/single_agent.py]: Hmm yeah okay. Perhaps you will have to rebase your `smarts_ppo_agent`  branch onto the `smarts_use_ultra_ppo_agent_from_develop` branch and then submit your changes.

Or, you can create a new branch, branching from `smarts_use_ultra_ppo_agent_from_develop`, and then apply the changes and create a new pull request.

Let me know if I anything was confusing or if I can help in any way.

**Pull request closed without merging**